frontend/src/modules/Metadata_Forms/components/MetadataFormFields.js

```diff
@@ -261,16 +261,19 @@ const DisplayTable = (props) => {
             Possible Values or Glossary Term
           </TableCell>
           <TableCell sx={{ width: '20px', alignContent: 'center' }}>
-            <Button
-              color="primary"
-              startIcon={<PencilAltIcon size={15} />}
-              sx={{ mt: 1 }}
-              onClick={startEdit}
-              type="button"
-              variant="outlined"
-            >
-              Edit
-            </Button>
+            {(userRole === userRolesMF.Owner ||
+              userRole === userRolesMF.Admin) && (
```

Comment:
what would be difference between Owner and Admin in this case? I see currently we are not resolving any user role as Admin but maybe you intend to in a future PR?

Reply:
I didn't decide yet. So, just in case I decided to make it the same way as for e.g. datasets.
If it's not needed, I will reduce it later.

Reply:
I give it another thought and removed Admin role for now.